pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -848,6 +848,64 @@ func newCertRotationController(
 	)
 	ret.certRotators = append(ret.certRotators, certRotator)
 
+	// add etcd cert
+	certRotator = certrotation.NewCertRotationController(
+		"etcdBackupServerCert",
```

Comment:
I'm not sure if this certificate should by managed by `kas-o`

Alternative would be to:
1. create your own CA and use it to generate the client cert
2. use CSR api to get a client certificate

Both `1` and `2` can be done in `etcd-o` operator.